index.d.ts

```diff
@@ -8,6 +8,8 @@ declare module 'posthog-node' {
         host?: string
         api_host?: string
         enable?: boolean
+        personalApiKey?: string
+        featureFlagsPollingInterval?: number
```

Comment:
Since existing options are `snake_case`, I think we should stick to that, even though it's a bit unusual for JS

Reply:
Only `api_host` is - `flushAt` and `flushInterval` aren't. Happy to put another pull request to support `apiHost` and deprecate `api_host`. This lib is still small enough that we can make better decisions and not end up with the `posthog-js` behemoth

Reply:
Actually, `api_host` hasn't been used for a while - it's only lingering here in the `d.ts` file, will remove it.